demo/pubsub/server/dynamic/subscribe_server.go

```
@@ -0,0 +1,96 @@
+}
+
+func main() {
+	flag.Parse()
+	if storeName == "" {
+		panic("storeName is empty.")
+	}
+	testDynamicSub()
+}
+
+func testDynamicSub() {
+	cli, err := client.NewClient()
```

golangci-lint

ineffectual assignment to err

(ineffassign)